data/map.txt

```diff
@@ -27844,6 +28041,22 @@ planet Lichen
 		threshold 3000
 		fleet "Large Militia" 8
 
+planet Lightstead
+	attributes station hai retirement
+	landscape land/space6
+	description `A visitor to this station will immediately notice that the gravity is lower than most planets by about a third. The reception area is lavish with several artworks and exotic plants. A well-groomed Hai receptionist informs you that, unless you have family at this respite facility, you will not be permitted to wander the station.`
```

Comment:
```suggestion
	description `A visitor to this station will immediately notice that the gravity is lower than most planets by about a third. The reception area is lavished with several artworks and exotic plants. A well-groomed Hai receptionist informs you that, unless you have family at this respite facility, you will not be permitted to wander the station.`
```

Reply:
I do not believe this change would be correct.

Reply:
Actually neither is correct. :)

```suggestion
	description `A visitor to this station will immediately notice that the gravity is lower than most planets by about a third. The reception area is bedecked with many works of art and exotic plants. A well-groomed Hai receptionist informs you that this is a respite home for the elderly, and unless you have family here, you will not be permitted to wander the station.`
```